.github/workflows/publish.yml

```diff
@@ -0,0 +1,51 @@
+name: Publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      package:
+        description: The package to release, this should be the path segment after packages/, without the @sapphire/ prefix
+        required: true
+
+jobs:
+  PublishPackage:
+    name: Publish @sapphire/${{ github.event.inputs.package}}
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'sapphiredev'
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: main
+          token: ${{ secrets.SKYRA_TOKEN }}
+      - name: Use Node.js v20
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: yarn
+          registry-url: https://registry.yarnpkg.com/
+      - name: Install Dependencies
+        run: yarn --immutable
```

Comment:
It never was, not for v3 and not even for v1 (as --frozen-lockfile). Yarn has always checked for `process.env.CI`. Doesn't take away that it's a good extra security guard to add.

Reply:
...Yes, it does, and I have CI proof from work to show it. This is redundant

Reply:
See https://v3.yarnpkg.com/cli/install#:~:text=or%20switching%20branches.-,If%20the,option%20is%20set%20(defaults%20to%20true%20on%20CI)%2C,-Yarn%20will%20abort:
![image](https://github.com/sapphiredev/utilities/assets/4019718/cfcc89ca-6f1a-49dd-ba5a-2a155b564f72)

And okay it was required for Yarn v1, I stand corrected. I haven't used Yarn v1 in forever though. 

![image](https://github.com/sapphiredev/utilities/assets/4019718/9511209e-b6a3-42c4-8050-fdaf7428766f)


Either way, we still provide the flag in every single sapphire repo in every single install command so that's cleanup for later